src/documents/TextToSpeechAPI.md

```diff
@@ -0,0 +1,580 @@
+---
+title: Text To Speech API
+layout: default
+---
+
+# Text To Speech API #
+
+The **Text To Speech** component uses [Web Speech Synthesis API](https://dvcs.w3.org/hg/speech-api/raw-file/tip/speechapi.html#tts-section) to queue up and read texts.
+
+## Browser Support ##
+The Text To Speech component can be used in browsers that support [Web Speech Synthesis API](https://dvcs.w3.org/hg/speech-api/raw-file/tip/speechapi.html#tts-section) is supported. At the moment when this document is written, Mar 4 2015, these browsers include:
+* Chrome 33+ 
+* Chrome for Android 40+
+* Safari 7.1+
+
+## Creator ##
+
+Use the following function to create a Text To Speech component:
+
+<table>
+    <tbody>
+        <tr>
+            <th>Method</th>
+            <td>
+                <code>fluid.textToSpeech(options);</code>
+            </td>
+        </tr>
+        <tr>
+            <th>Description</th>
+            <td>
+                Instantiate the text to speech component. The methods of the instantiated component can be called to queue up and read texts.
+            </td>
+        </tr>
+        <tr>
+            <th>Parameters</th>
+            <td>
+                <dl>
+                    <dt>options</dt>
+                    <dd>
+                        An optional data structure that configures the Text To Speech component, as described below.
+                    </dd>
+                </dl>
+            </td>
+        </tr>
+        <tr>
+            <th>Returns</th>
+            <td>The Text To Speech component</td>
+        </tr>
+        <tr>
+            <th>Examples</th>
+            <td>
+<pre>
+<code>
+var tts = fluid.textToSpeech({
+    utteranceOpts: {
+        volume: 1
+    }
+});
+</code>
+</pre>
+            </td>
+        </tr>
+        <tr>
+            <th>See also</th>
+            <td>
+                <a href="#utteranceopts-options">Utteranceopts Options</a>
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+## Supported Methods ##
+
+### `queueSpeech` ###
+
+<table>
+    <tbody>
+        <tr>
+            <th>Method</th>
+            <td>
+                <code>textToSpeech.queueSpeech(text, interrupt, options);</code>
+            </td>
+        </tr>
+        <tr>
+            <th>Description</th>
+            <td>
+                The <code>queueSpeech</code> method allows you to append the text to the end of the queue to begin speaking after the other texts in the queue have been spoken. This method can also be used to remove all texts from the queue by passing in the `interrupt` argument with a true value.
```

Comment:
This last sentence might be better worded as "Setting the 'interrupt' argument to true will remove all texts from the queue before adding 'queueSpeech'.